### PR TITLE
[JFG-971] put docker-compose files to the nexus

### PR DESCRIPTION
**Pull request by Nmishin**
Two submodules have been added, docker and compose. On the compose module we create compose package which include compose*.yaml and directory "files", package it to the zip archive and push to the nexus.

### CODE REVIEW

File: docker/pom.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <id>docker-resources</id>
+                            <phase>none</phase>
```
Comment (anton-antonenko) [inline docker/pom.xml]: why phase is none?

Comment (Nmishin) [inline docker/pom.xml]: because now we don't need enable it by default.

File: pom.xml [hunk 2]
```diff
@@ -49,6 +49,7 @@
 
     <modules>
         <module>dbapi</module>
+        <module>docker</module>
```
Comment (a-badaev) [inline pom.xml]: shouldn't its build be triggered only by a special profile?

Comment (Nmishin) [inline pom.xml]: I think no, because we want to put artefacts to nexus every time. 
We use a special profile for build and push docker images.

**Pull request merged**